DQM/Integration/python/clients/hcal_dqm_sourceclient-live_cfg.py

```diff
@@ -32,10 +32,11 @@
 #-------------------------------------
 from DQM.Integration.config.online_customizations_cfi import *
 if useOfflineGT:
-	process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
-	process.GlobalTag.globaltag = autoCond['run3_data_prompt'] 
+        process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+        process.GlobalTag.globaltag = '141X_dataRun3_Prompt_Candidate_2024_10_08_09_42_50'
```

Comment:
none of the comments of the 14.1.X backport was addressed.

Reply:
I opted to get this up such that it was faithful to the backport version code - happy to address the comments now, will go thru and recommit

Reply:
Adding: I will go thru and update this PR and the backport simultaneously

Reply:
Responded issue-by-issue in backport thread, all suggestions adopted, both PRs updated